packages/api/lib/writeRecords/write-granules.js

```diff
@@ -747,14 +749,14 @@ const writeGranuleFromApi = async (
     // Validate fields that cannot/shouldn't be null aren't
     const invalidNullableFields = {
       status,
-      createdAt,
       updatedAt,
       granuleId,
       collectionId,
       execution,
+      producerGranuleId,
     };
     Object.entries(invalidNullableFields).forEach(([key, field]) => {
-      if (isNull(invalidNullableFields[field])) {
+      if (isNull(field)) {
```

Comment:
check the value of each field.

Reply:
*frown* that's an unfortunate bug you just fixed, good catch.  

We shouldn't be expecting NULL for created at, right?      Remember that this handles both PATCH and PUT, and in the case of PATCH there's no default value *if* it's not new, by design. 

What we really don't want to do is allow a NULL value for a required/preset field. 

:thinking:

Reply:
If createdAt and updatedAt can't be null, for PUT request, we have to provide updatedAt and createdAt, otherwise the request will fail.

Reply:
*nods* I think they're being provided upstream in L286 (for new granule patch).   PUT uses nulls for everything not passed in (L357), so it should fail validation if anything that's required isn't specified.

Just re-stating to make sure we're both in agreement re: the intent of the current code (I think we are?)     If the above doesn't track let's meet/walk through it?